apps/api/src/report/report.service.ts

```diff
@@ -765,6 +770,20 @@ export class ReportService {
     return data;
   }
 
+  /**
+   *
+   * @param dates start and end date for data extract YYYY-MM-DD
+   * @returns
+   */
+  async extractMilestoneData(dates?: ReportPeriodDTO) {
+    const { from, to } = dates || {from:'2001-01-01', to:dayjs().toISOString()};
```

Comment:
It might not make any difference, but to match the pattern, how about `dayjs().format('YYYY-MM-DD')`?